src/index.tsx

```diff
@@ -72,14 +82,13 @@ interface State {
   lastDrag: Drag | null;
   heights: { [key: string]: HeightData } | null;
 }
-export default class DraggableList<
+export class DraggableList<
   I,
   C,
-  T extends React.Component<Partial<TemplateProps<I, C>>>
+  T extends React.Component<Partial<TemplateProps<I, C, T>>>
```

Comment:
I think this bound on T no longer makes sense with the render props style. T can be anything at all now, so drop the whole `extends ...` bit here and in the other components with `T` type parameters.

Reply:
[054e748](https://github.com/StreakYC/react-draggable-list/pull/62/commits/054e7481c01d1a2febdd4cb81b4ce9e1b959daac)